undefined/index.nunj

```diff
@@ -0,0 +1,19 @@
+{% extends "layouts/default.nunj" %}
+
+{% set title = "Walmark" %}
+
+{% block content %}
+    {% include "../tpl/components/greetings.nunj" %}
```

Comment:
This relative path is sometimes hard to read, if you are deep in the structure.
This should work, but depends on your structure: `{% include "components/greetings.nunj" %}`